lib/symboldatabase.cpp

```diff
@@ -7089,7 +7089,14 @@ ValueType::MatchResult ValueType::matchParameter(const ValueType *call, const Va
 
 ValueType::MatchResult ValueType::matchParameter(const ValueType *call, const Variable *callVar, const Variable *funcVar)
 {
-    ValueType::MatchResult res = ValueType::matchParameter(call, funcVar->valueType());
+    ValueType vt;
+    auto pvt = funcVar->valueType();
```

Comment:
Maybe do `ValueType vt = *funcVar->valueType();` and then just modify it directly.

Reply:
We have to check for nullptr first, otherwise there is a crash when running the tests.